pulumi/aws/anthos/ingress/boa-ingress.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
The api version for this config is now at: `networking.k8s.io/v1`. If we could move off of beta, it would be good.

Reply:
#26 has been opened to track this; I will circle back to this after we have anthos merged to master.